apps/studio/components/layouts/ProjectLayout/ProjectLayout.tsx

```diff
@@ -168,18 +195,23 @@ const ProjectLayout = forwardRef<HTMLDivElement, PropsWithChildren<ProjectLayout
                 </MenuBarWrapper>
               </ResizablePanel>
               <ResizableHandle
-                className={cn({ hidden: !showProductMenu || !productMenu })}
+                className={cn('hidden md:flex', { '!hidden': !showProductMenu || !productMenu })}
                 withHandle
                 disabled={resizableSidebar ? false : true}
               />
               <ResizablePanel id="panel-right" className="h-full flex flex-col">
-                {!hideHeader && IS_PLATFORM && <LayoutHeader />}
+                {!hideHeader && IS_PLATFORM && (
+                  <LayoutHeader
+                    showProductMenu={!!(showProductMenu && productMenu)}
```

Comment:
can we stop using props for things like this?
should we not be using app state?
@joshenlim @fsansalvadore 

I'm in another branch and I'm probably going to remove these props if that's ok.

Reply:
also title, could have logic for the title handled inside component